WallPaper_Project1/WallPaper_Project1/Data/Source/Local/CoreData.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  CoreData.swift
+//  WallPaper_Project1
+//
+//  Created by DuyThai on 31/01/2023.
+//
+
+import Foundation
+import UIKit
+import CoreData
+
+struct CoreData {
+    static let shared = CoreData()
+    private var appDelegate: AppDelegate?
+
+    private init() {
+        appDelegate = UIApplication.shared.delegate as? AppDelegate
+    }
+
+    func addToCoreData(data: CoreDataObject?, completion: @escaping ( String?) -> Void) {
```

Comment:
Lỗi coding convention
```suggestion
    func addToCoreData(data: CoreDataObject?, completion: @escaping (String?) -> Void) {
```